.github/workflows/prerelease.yml

```diff
@@ -31,10 +34,10 @@ jobs:
       - name: Generate new versions
         id: generate-versions
         run: |
-          preid=${{ github.ref_name }}
-          normalized_pre_id=$(echo "$preid" | tr -dc '[:alnum:]\n\r' | tr '[:upper:]' '[:lower:]')
+          commit_sha=$COMMIT_SHA
+          preid="test-${commit_sha:0:5}"
```

Comment:
Is it common to use the word "test" for prerelease versions? I think it is a bit weird

Reply:
It's not common, the most common pre-releases names are `alpha`, `beta` and `rc`. According to the [semver](https://semver.org/#spec-item-9) specs it's valid to have any alphanumeric values, so it's up to us. I think it's way better than using the branch name, especially when it can become long, unclear, and confusing. We can decided on some other structure but I don't think the branch name is a good choice
![Screenshot 2024-08-14 at 14 50 40](https://github.com/user-attachments/assets/8653debb-d3e8-4db3-95ea-02f7c8678ed2)

Reply:
@shaharzil changed to `alpha`